config_builder.go

```diff
@@ -1420,3 +1425,41 @@ func parseHeaderStateAssertion(state string) (*headerfs.FilterHeader, error) {
 		FilterHash: *hash,
 	}, nil
 }
+
+// broadcastErrorMapper maps errors from bitcoin backends other than neutrino to
+// the neutrino BroadcastError which allows the Rebroadcaster which currently
+// resides in the neutrino package to use all of its functionalities.
+func broadcastErrorMapper(err error) error {
+	returnErr := wallet.MapBroadcastBackendError(err)
+
+	// We only filter for specific backend errors which are relevant for the
+	// Rebroadcaster.
+	var errAlreadyConfirmed *wallet.ErrAlreadyConfirmed
+	var errInMempool *wallet.ErrInMempool
+	var errMempoolFee *wallet.ErrMempoolFee
+
+	switch {
+	// This makes sure the tx is removed from the rebroadcaster once it is
+	// confirmed.
+	case errors.As(returnErr, &errAlreadyConfirmed):
```

Comment:
I guess this could be "Is" and we do not have to create the variable?

Reply:
Also not sure here. I think for `Is` we'd compare values and for `As` we'd compare types, so I guess `As` is the right choice here? Anyway in either case we'd need the variables tho